core/package.json

```diff
@@ -7,7 +7,7 @@
   },
   "scripts": {
     "preinstall": "npx only-allow pnpm",
-    "postinstall": "cd .. && husky install && cd core/packages/test && ./scripts/hack-ethereum.sh",
+    "postinstall": "cd .. && husky install && cd core/packages/test",
```

Comment:
I am removing the `hack-ethereum.sh` script, because deviating from the official [Ethereum minimal spec](https://github.com/ethereum/consensus-specs/blob/ffacbb442b8d4469e857a79dee6932fb2420cf89/configs/minimal.yaml) gave me some issues with the ancestry proofs and I believe in this case to stick to the standard (at the cost of slightly slower tests), make sense.

Reply:
I agree, it complicates testing.